### ISSUE TITLE
Would the image based gym code also be released? 

### ISSUE THREAD

**Issue by Wangweiyao** [elicassion/StARformer#1]
Hi, thanks for open source the codebase. It looks to me that the gym code is using the state directly (not from image). https://github.com/elicassion/StARformer/blob/main/gym/models/starformer.py#L266
Would the image based gym code also be released? Thanks

**Comment by elicassion** [comment]
Hi @Wangweiyao,

Thanks for your interests and the kind reminder! I have updated the code for DMC w/ image input in `atari_and_dmc/run_star_dmc.py`.  (Note: I changed the directory name from `atari` to `atari_and_dmc`)

To run on DMC, you will need to install `dm-control` and `mujoco` dependencies. I have updated the environment file `atari_and_dmc/conda_env.yml` and the corresponding instructions in README.

Basically the usage is similar to atari as they both take image input. The only differences are data loading and things related to action prediction head (discrete / continuous). Ideally you can unify two files (`run_star_dmc.py` and `run_star_atari.py`) in one, but here I am providing two due to its clearness and easiness for editing.

Please let me know if you have any further questions 🌟
